indigo/analysis/toc/base.py

```diff
@@ -368,17 +368,13 @@ def as_dict(self):
             'title': self.title,
             'children': [c.as_dict() for c in self.children],
             'basic_unit': self.basic_unit,
+            'num': self.num,
```

Comment:
should this be self.num or ''?

Reply:
no, this is fine, it must be None if not present not `''`

Reply:
I think add heading here too.